src/MayaraClient.cpp

```
@@ -108,6 +109,53 @@ std::string WsUrl(const std::string& base, const std::string& radar_id) {
+
+// Is this an answer from the radar API, or merely a 200 from a server that
+// happens to route that path? A Signal K server without mayara installed is
+// the case that matters: reaching Signal K is no proof that the radar plugin
+// is there, and treating the two alike made the plugin settle on a server that
+// could never produce a radar. Every shape we accept carries either the 3.4.0
+// envelope or at least one radar; a bare {} carries neither and is rejected.
+bool LooksLikeRadarApi(const json& j) {
+  if (j.is_array()) return true;
+  if (!j.is_object()) return false;
+  if (j.contains("radars") && j["radars"].is_object()) return true;
+  if (j.contains("version") && j["version"].is_string()) return true;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)